corehq/apps/hqwebapp/tests/test_views.py

```diff
@@ -144,6 +144,7 @@ def setUpClass(cls):
     def tearDownClass(cls):
         cls.user.delete(cls.domain, deleted_by=None)
         cls.project.delete()
```

Comment:
It would be safer to use `cls.addClassCleanup` to delete these.

Reply:
https://github.com/dimagi/commcare-hq/pull/33592/commits/119d15c5c54623e2ceb32a2091801a1751272811